torch/csrc/jit/ir/ir.cpp

```diff
@@ -104,7 +104,7 @@ std::ostream& operator<<(
 static void printAttribute(std::ostream& out, const at::Tensor& tensor) {
   // 1-elem tensors are usually boxed scalars, so print them like it
   if (tensor.numel() == 1) {
-    auto scalar_tensor = tensor.view({}).item();
+    auto scalar_tensor = tensor.view(std::vector<int64_t>{}).item();
```

Comment:
Is there a way to stop `{}` from being resolved to `ScalarType`?

Reply:
Do we still need these changes if dtype overload is blocklisted?

Reply:
Yes, it is blocklisted at runtime for schema matching, but not at compile time as in here.